gulp/util.js

```diff
@@ -48,6 +49,9 @@ function buildJs () {
 
   var jsBuildStream = gulp.src( jsFiles )
       .pipe(filterNonCodeFiles())
+      .pipe(babel({
+          presets: [["env", {targets: {browsers: config.browsers}}]]
```

Comment:
I think `uglify: true` should be added here. [Readme](https://github.com/babel/babel-preset-env#targetsuglify) and [PR](https://github.com/babel/babel-preset-env/pull/178).